redisinsight/ui/src/pages/rdi/components/header/Header.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiToolTip
+} from '@elastic/eui'
+import cx from 'classnames'
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+
+import { Pages } from 'uiSrc/constants'
+import { connectedInstanceSelector } from 'uiSrc/slices/rdi/instances'
+
+import styles from './styles.module.scss'
+
+export interface Props {
+  actions: JSX.Element
+}
+
+const Header = ({ actions }: Props) => {
```

Comment:
Can you take a look at this component? `RdiPipelineHeader` i guess we can combine them in one component

Reply:
I will take a look